modules/swiss/src/main/BsonHandlers.scala

```diff
@@ -106,7 +106,7 @@ object BsonHandlers:
         "c"  -> (s.chatFor != Swiss.ChatFor.default).option(s.chatFor),
         "i"  -> s.roundInterval.toSeconds.toInt,
         "p"  -> s.password,
-        "o"  -> s.conditions.nonEmpty.option(s.conditions),
+        "o"  -> s.conditions,
```

Comment:
Also tried `w.zero(s.conditions)` but after looking at the result in the db was the same.